recipe/meta.yaml

```
@@ -1,44 +1,79 @@
-{% set name = "flask-session" %}
-{% set version = "0.4.0" %}
+{% set name = "Flask-Session" %}
+{% set version = "0.8.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-Session-{{ version }}.tar.gz
-  sha256: c9ed54321fa8c4ca0132ffd3369582759eda7252fb4b3bee480e690d1ba41f46
+  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower|replace('-', '_') }}-{{ version }}.tar.gz
+    sha256: 20e045eb01103694e70be4a49f3a80dbb1b57296a22dc6f44bbf3f83ef0742ff
+  - url: https://github.com/pallets-eco/{{ name|lower }}/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 017c852d3df6b8492666b5e3544bb4f283a49ca357fac28af465b8cb8017f1e4
+    folder: gh
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  skip: True  # [py<37]
+  skip: true  # [py<38]
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
-
+    - flit-core <4
   run:
     - python
-    - cachelib
-    - flask >=0.8
+    - cachelib >=0.10.2
+    - flask >=2.2
+    - msgspec >=0.18.6
+  run_constrained:
+    - pymongo >=4.6.2
+    - redis-py >=5.0.3
+    - flask-sqlalchemy >=3.0.5
```

https://github.com/pallets-eco/flask-session/blob/0.8.0/pyproject.toml#L30
```suggestion
    - flask-sqlalchemy >=3.0.5
    - cachelib >=0.10.2
```